instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/PeerServiceAttributesExtractor.java

```diff
@@ -67,10 +67,6 @@ public void onEnd(
 
     String peerName = attributesGetter.peerName(request, response);
     String peerService = mapToPeerService(peerName);
-    if (peerService == null) {
-      String peerIp = attributesGetter.peerIp(request, response);
-      peerService = mapToPeerService(peerIp);
-    }
```

Comment:
alternatively we could check both `net.sock.peer.addr` and `net.sock.peer.name`, but now that we are putting IP address into `net.peer.name` when there's no hostname I don't think there is a need(?)